metrics/metrics.go

```diff
@@ -17,26 +16,24 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-type DKGStatus string
-type ReshareStatus string
-
-const UnknownBeaconID = "unknown"
+type DKGState int
+type ReshareState int
 
 const (
-	DKGNotStarted    DKGStatus = "not_started"
-	DKGInProgress    DKGStatus = "in_progress"
-	DKGWaiting       DKGStatus = "waiting"
-	DKGReady         DKGStatus = "ready"
-	DKGUnknownStatus DKGStatus = "unknown"
-	DKGShutdown      DKGStatus = "node_stopped"
+	DKGNotStarted   DKGState = 0
+	DKGWaiting      DKGState = 1
+	DKGInProgress   DKGState = 2
+	DKGDone         DKGState = 3
+	DKGUnknownState DKGState = 4
+	DKGShutdown     DKGState = 5
```

Comment:
You could use `iota` here:
```suggestion
	DKGNotStarted   DKGState = iota
	DKGWaiting
	DKGInProgress
	DKGDone
	DKGUnknownState
	DKGShutdown
```

Reply:
I'd rather make it explicit since the IDs are referenced in the help message for the metric.